### PR TITLE
Tag Plots 0.7.0, RecipesBase 0.0.2, and VisualRegressionTests 0.0.4

### PR DESCRIPTION
**Pull request by tbreloff**
This is a huge (somewhat breaking) update/improvement for the the Plots ecosystem.  See [the NEWS](https://github.com/tbreloff/Plots.jl/blob/master/NEWS.md) and the linked [summary](http://plots.readthedocs.io/en/latest/plots_v0.7/) for the full scoop.

As always, please file issues with bugs or feature requests, and swing by the gitter channel with questions.  The docs are slowly filling out, but I still have a lot to do there (help would be appreciated, though you have to understand Plots to do it ;)  I'll post on julia-users in the next week after I've had a chance to work through a few more bugs and add more documentation.

### CODE REVIEW

**Comment by tkelman** [comment]
How breaking? If users run Pkg.update will getting this version of Plots break other packages that use it? If so, then you should add upper bounds on the Plots requirements of past versions of those packages.

**Comment by tkelman** [comment]
RecipesBase is failing its tests, is that expected?

**Comment by tbreloff** [comment]
Hmm. Any breakage will depend on how they use Plots. I'll check it out and
report back. Of course, one key point is that most packages should stop
depending on Plots, and switch to adding recipes to RecipesBase. I'll work
with package owners to make the shift.

As for the RecipesBase tests... Yeah I guess that's expected. I'll update
to get the green, though it's well tested in Plots.

**Comment by tbreloff** [comment]
I pushed up fixes for RecipesBase's tests... they pass now.  I guess I'll re-tag, even though I only changed tests.

As for Plots breakage... there's a lot more packages depending on Plots than I realized!  I'll open an issue to track it and cc you.

File: ExperimentalAnalysis/versions/0.0.2/requires [hunk 1]
```diff
@@ -1,5 +1,5 @@
 julia 0.4
 DataFrames
-Plots
+Plots 0.0 0.7
```
Comment (tkelman) [inline ExperimentalAnalysis/versions/0.0.2/requires]: does 0.0.1 also need this?

**Comment by tbreloff** [comment]
Doh! I labeled the commit message wrong.  I actually updated ExperimentalAnalysis in that last commit, not EEG.  Sorry.

**Comment by tbreloff** [comment]
I think this is good to merge.  I'll be on standby to fix anything unexpected that pops up.

**Comment by tbreloff** [comment]
Thanks @tkelman!

**Pull request merged**